### DISCUSSION TITLE
VDH - MKV download instead of MP4

### DISCUSSION THREAD

**Discussion by kewpieaddict** [aclap-dev/video-downloadhelper#77] [Help and Q&A]
### Your question

I badly need help. When I used VDH 8.2.0.19 on my new laptop, the video started downloading as an MKV file instead of MP4. But when I use VDH (older version 8.1.6.0) on my gaming PC, it downloads an MP4 file which I prefer. 

I need MP4 files, not MKV. How do I change this MKV file to MP4 in VDH? I don't want to use the converter, it takes too much time. I just want it to work smoothly just like on my gaming PC. 

I use it on chrome.

**Comment by GrampaWildWilly** [comment]
The Addon & CoApp version and details section is there for a reason.  Please provide that information.  Read this to find out how: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by paulrouget** [comment]
I need the URL of the video.

**Comment by paulrouget** [comment]
Ok, I figured it out. Working on a fix. Should be part of v8.2.0.22, tomorrow or so.
